docs/migration/migration1100to1110.adoc

```diff
@@ -0,0 +1,19 @@
+= v1.10.0 → v1.11.0
+
+coff:ee v1.10.0 -> v1.11.0 migrációs leírás, újdonságok, változások leírása
+
+== Változások
+
+=== Általános
+
+==== BOM verzió emelések:
+
+=== coffee-rest
+* Bekerült egy defaultként használható OpenApiFilter amit a következő configuráció segítségével lehet aktiválni:
+[source,properties]
+----
+mp.openapi.filter=hu.icellmobilsoft.coffee.rest.filter.OpenAPIFilter
```

Comment:
ez a leiras a rendes leirasok kozott kell lennie. Az hogy itt van a migracioban egy bonusz csak, de kozben a kovetkezo verziokkal meg valtozhat, tehat a linkeles kellene a valodi tartalomra.

Reply:
Igen közben eszembe jutott, és már meg is csináltam, tettem bele olyat is hogy miként lehet leszármaztatni, példával.